stripe/src/main/java/com/stripe/android/paymentsheet/DefaultPaymentSheetLauncher.kt

```diff
@@ -22,7 +25,56 @@ internal class DefaultPaymentSheetLauncher(
 
         // lazily access the statusBarColor in case the value changes between when this
         // class is instantiated and the payment sheet is launched
-        { activity.window.statusBarColor }
+        { getStatusBarColor(activity) }
+    )
+
+    constructor(
```

Comment:
Are you really using all of these constructors?

Reply:
Good point, we don't need all of them right now!